pnpm-workspace.yaml

```diff
@@ -10,6 +10,7 @@ packages:
 catalog:
   '@reduxjs/toolkit': ^2.8.2
   immer: ^10.1.1
+  msw: ^2.5.1
```

Comment:
👍 do you think we should update the device-client package.json to use the catalog also then?